specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlPool.json

```diff
@@ -8015,7 +8015,7 @@
         },
         "stepIndex": {
           "description": "dataWarehouseQueries step index",
-          "type": "string",
+          "type": "integer",
```

Comment:
Specify the format here as required in https://github.com/Azure/azure-openapi-validator/blob/main/docs/integer-type-must-have-format.md to fix the lint error. You can find the example in `bigDataPool.json` in the current PR.

Reply:
@mkagithala Please resolve this comment to fix the lint error.